packages/shared-ui/README.md

```
@@ -0,0 +1,27 @@
+> [!IMPORTANT]
+> This is an internal package. It may change without warning and is not subject to semantic versioning. Use at your own risk.
+
+A collection of shared UI components for UCD.js projects.
+
+## Installation
+
+```bash
+npm install @ucdjs-internal/shared-ui
+```
+
+  ## 📄 License
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)